ethpm/utils/manifest_validation.py

```diff
@@ -114,3 +115,9 @@ def validate_manifest_exists(manifest_id: str) -> None:
         raise ValidationError(
             "Manifest not found in V2_PACKAGES_DIR with id: {0}".format(manifest_id)
         )
+
+
+def format_manifest(manifest: Manifest, prettify: bool) -> str:
```

Comment:
You might change this signature to be `format_manifest(manifest, *, prettify)` so that `prettify` is required to be passed in as a keyword argument.